.github/workflows/contract-size.yml

```diff
@@ -8,7 +8,13 @@ jobs:
     strategy:
       matrix:
         package:
-          [core-contracts, core-modules, synthetix-main, spartan-council, governance-modules]
```

Comment:
Moved generic v3gm packages to core-modules, and renamed governance-modules (which was supposed to be the generic one but had SNX specific "stuff" in it) to synthetix-governance.